packages/schematics/angular/secondary-entrypoint/index.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {
+  Rule,
+  SchematicsException,
+  Tree,
+  apply,
+  applyTemplates,
+  chain,
+  mergeWith,
+  move,
+  strings,
+  url,
+} from '@angular-devkit/schematics';
+import { JSONFile } from '../utility/json-file';
+import { latestVersions } from '../utility/latest-versions';
+import { relativePathToWorkspaceRoot } from '../utility/paths';
+import { buildDefaultPath, getWorkspace } from '../utility/workspace';
+import { ProjectType } from '../utility/workspace-models';
+import { Schema as LibraryOptions } from './schema';
+
+function updateTsConfig(packageName: string, ...paths: string[]) {
+  return (host: Tree) => {
+    if (!host.exists('tsconfig.json')) {
+      return host;
+    }
+
+    const file = new JSONFile(host, 'tsconfig.json');
+    const jsonPath = ['compilerOptions', 'paths', packageName];
+    const value = file.get(jsonPath);
+    file.modify(jsonPath, Array.isArray(value) ? [...value, ...paths] : paths);
```

Comment:
The path of the new entry-point, should probable be added right after of the existing entries of the same library.